src/libraries/System.Text.Json/tests/Common/CollectionTests/CollectionTests.AsyncEnumerable.cs

```diff
@@ -16,6 +16,7 @@ public abstract partial class CollectionTests
     {
         [Theory]
         [MemberData(nameof(GetAsyncEnumerableSources))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/86973", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
```

Comment:
This is fixable - add this to the project file:

https://github.com/dotnet/runtime/blob/349bbe938c420af98909c266b53092dd3ae15e13/src/libraries/System.Runtime/tests/System.Runtime.Tests.csproj#L26-L28

Then create the XML file and put the required generic instantiations in it, like this:

https://github.com/dotnet/runtime/blob/ed2634403de655ba68230ad531e2b7cb1c21b87f/src/libraries/System.Runtime/tests/default.rd.xml#L45-L88

Reply:
Up to you but this is literally what we did in every test that we're running. We don't have any test in the product disabled because of this.

Reply:
https://github.com/dotnet/runtime/pull/83933 has been working towards fixing this. The problem is in xunit.

Reply:
Pushed new changes that should enable testing for all remaining generic theories.